Cargo.toml

```diff
@@ -22,9 +22,11 @@ travis-ci = { repository = "jonhoo/hdrsample" }
 [features]
 #benchmark = ["criterion"]
 benchmark = [] # for crates.io publication
+bench_private = [] # for enabling nightly-only feature(test) on the main crate to allow benchmarking private code
```

Comment:
Argh, it doesn't work, because then it tries to use criterion and can't find the dependency. :(